tools/docker/Dockerfile

```diff
@@ -99,4 +99,4 @@ RUN if [ ! -n "${MIN_RUN_ENV}" ]; then \
     fi && \
     rm -rf /var/lib/apt/lists/* /root/*.deb
 USER ${USERNAME}
-WORKDIR /home/${USERNAME}
+WORKDIR /home/${USERNAME}
```

Comment:
uh....

you may need to clean up commits..

Reply:
Done